.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
```

Comment:
```suggestion
name: Idea box
```

Maybe we should some other wording here instead of `feature request`. 😄 
Maybe something like:

- `Idea box`
- `Enhancement idea`
- `Enhancement suggestion`
- `Suggest improvement`

Maybe @ben has a good idea for how we should call this?